rb/lib/selenium/webdriver/common/driver.rb

```diff
@@ -71,7 +71,7 @@ def for(browser, opts = {})
       def initialize(bridge: nil, listener: nil, **opts)
         @devtools = nil
         @bidi = nil
-        bridge ||= create_bridge(**opts)
+        bridge ||= create_bridge(caps: opts[:caps], url: opts[:url], http_client: opts[:http_client])
```

Comment:
I feel like there could be more in `opts` than listed. Is there any way to keep using kwargs?

Reply:
I agree, I tried different variations in the signature and to overload it but I kept getting the same error,
this is why several methods that use kwargs are ignored in the steepfile, I found the same issue here:

https://github.com/soutaro/steep/issues/429

And I don't think the fix is actually working, I will roll this back and add the file to ignore as with the other kwargs issues

Reply:
Should we define the signature as?

```ruby
def create_bridge: (Hash[Symbol, String]) -> untyped
```

Reply:
The issue is that by doing that, the other uses of create_bridge, I get a MethodArityMismatch error and MethodParameterMismatch

it's the same if I put both values as nillable or if I overload the method declaration

I need to investigate more on the kwargs implementation because is not the only place that I have this issue

<img width="1324" alt="Screenshot 2024-03-16 at 00 28 28" src="https://github.com/SeleniumHQ/selenium/assets/33221555/b31483b6-e05f-4178-a4b7-09ea3d0e9ea7">